test/vanilla/AcceptanceTests/test_invoke.py

```diff
@@ -0,0 +1,106 @@
+# --------------------------------------------------------------------------
```

Comment:
added tests for invoke. both vanilla and azure tests. vanilla tests deal with requests with body, and stream